colibre/auto_plotter/cold_gas_relations.yml

```diff
@@ -261,7 +263,7 @@ stellar_mass_HI_mass_30:
   metadata:
     title: Stellar Mass-HI Gas Mass (30 kpc Stellar Mass)
     caption: All galaxies are included in the median line.
-    section: Cold Gas Relations
+    section: Original Cold Gas Relations
```

Comment:
The name original doesn't really make very much sense here. I guess all of these are against gas mass? Can we do `Cold Gas Relations (Gas)` or `Cold Gas Relations ($M_{\rm g}$)`?